frontend/src/metabase-types/api/mocks/database.ts

```diff
@@ -37,6 +37,9 @@ export const createMockDatabase = (opts?: Partial<Database>): Database => ({
   native_permissions: "write",
   initial_sync_status: "complete",
   features: COMMON_DATABASE_FEATURES,
+  uploads_enabled: false,
+  uploads_schema_name: null,
+  uploads_table_prefix: null,
```

Comment:
FE reviewers: these new columns are not currently used on the FE, the settings are. Instead these columns are used on the BE to store the data.